include/TPP/Dialect/Xsmm/XsmmUtils.h

```diff
@@ -88,6 +98,28 @@ ArrayAttr getUnaryDispatchFlags(UnaryOp op);
 
 ArrayAttr getBinaryDispatchFlags(BinaryOp op);
 
+int64_t getOredFlags(ArrayAttr flags);
```

Comment:
This already exists in `ConvertXsmmToFunc`, please reuse there.

Reply:
I did not want to add dependences between the two conversion modules, and we were planning to retire ConvertXsmmToFunc after this. But if you suggest that I reuse this function, I will for now, We can move it to xsmmutils when we retire it, I suppose.

Reply:
Not depending on ConvertXsmmToFunc now, because this function is no longer required.